index.php

```diff
@@ -0,0 +1,127 @@
+<?php
+?>
+
+<html lang="en-GB">
+
+	<head>
+		<title> Welcome!</title>
+		<link rel="stylesheet" type="text/css" href="style.css">
+	</head>
+
+<body>
+<nav class="nav_bar">
+    <p class="welcome">Welcome to my own personal Plant Database</p>
+    <div class="links_nav">
+        <a href="form.php">Add new plants to the database</a>
+    </div>
+</nav>
+
+<main>
+    <h1>Hi!</h1>
+    <h1>Here you can see my stunning house plant collection:</h1>
+
+    <div class="plants_collection">
+        <div class="plants">
+            <img class="prayer_plant" src="images/prayer_plant.jpg" alt="here you can see my stunning Prayer Plant or Calathea medallion">
+            <p>Prayer Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Calathea medallion</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Prayer plant</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial, indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>76.3 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>No</ul>
+
+        </div>
+
+        <div class="plants">
+            <img class="elephant_ear" src="images/elephant-ear-plant.jpg" alt="beautiful elephant ear plant with a blue background">
+            <p>Elephant Ear Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Alocasia zebrina</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Elephant ear plant</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial, indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>55.5 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>Yes</ul>
+        </div>
+
+        <div class="plants">
+            <img class="spider_plant" src="images/curly-spider-plant.jpg" alt="curly spider plant in a grey pot">
+            <p>Curly Spider Plant</p>
+            <h2>BOTANICAL NAME:</h2>
+            <ul>Chlorophytum comosum</ul>
+            <h2>NICKNAME:</h2>
+            <ul>Curly spider plant; Variegated spider plant; Airplane plant</ul>
+            <h2>PLANT TYPE:</h2>
+            <ul>Perennial indoor</ul>
+            <h2>PLANT HEIGHT:</h2>
+            <ul>30 cm</ul>
+            <h2>POISENOUS:</h2>
+            <ul>No</ul>
+
```

Comment:
is this space necessary? maybe remove so that we know this whole section of code is a div!

Reply:
done!